src/src/PluginStructs/P109_data_struct.cpp

```diff
@@ -289,10 +289,7 @@ bool P109_data_struct::plugin_once_a_second(struct EventStruct *event) {
  * Save thermo settings unconditionally
  *************************************************************************/
 void P109_data_struct::saveThermoSettings(struct EventStruct *event) {
-  String fileName;
-
-  fileName += concat(F("thermo"), static_cast<int>(event->TaskIndex + 1));
-  fileName += F(".dat");
+  const String fileName(strformat(F("thermo%d.dat"), static_cast<int>(event->TaskIndex + 1)));
```

Comment:
The `static_cast<int>` was only added to avoid having an extra template generated... (not sure if that was effective though)

Reply:
Well taskindex_t is of type unsigned char (uint8_t) and those almost char types sometimes might be seen as char instead of integer type.
So I will leave this. It will not make the build larger, just an excuse not to change it :)